_data/toc/insight-engine.yaml

```diff
@@ -39,6 +39,8 @@
           path: '/insight-engine/latest/config/filtered/'
         - title: 'Transactional metadata query'
           path: '/insight-engine/latest/config/transactional/'
+        - title: 'Performance Recommendations'
+          path: '/search-services/latest/config/performance/'
```

Comment:
Should this link go to search-services even though we are in insight-engine? The /insight-engine/latest/config/performance/ page does exist...

Reply:
no it shouldn't !